src/style/base/_root.scss

```diff
@@ -0,0 +1,15 @@
+:root {
+  --primary: rgb(0, 106, 245);
+  --text-primary-1: rgba(255, 255, 255);
+  --text-primary-2: rgba(255, 255, 255, 0.6);
+  --text-secondary-1: rgb(34, 34, 34);
+  --text-secondary-2: rgb(34, 34, 34, 0.6);
+  --table-header: rgb(34, 34, 34, 0.3);
+  --table-hover: rgb(34, 34, 34, 0.05);
+  --background-base: rgb(241, 241, 241);
+}
+
+:root .dark {
+  --background-base: rgb(0, 0, 0);
+  --text-primary-1: rgba(255, 255, 255);
+}
```

Comment:
Pay attention to all colors and their respective changes in light vs dark mode. Include all colors that are changing in dark class.